libcontainer/rootfs_linux.go

```diff
@@ -594,6 +595,37 @@ func createDevices(config *configs.Config) error {
 }
 
 func bindMountDeviceNode(dest string, node *configs.Device) error {
+	var st unix.Stat_t
+	if err := unix.Lstat(dest, &st); err != nil {
+		if !os.IsNotExist(err) {
+			return errors.Wrapf(err, "could not stat existing path for device node: %q", dest)
+		}
+	} else if os.FileMode(st.Mode)&os.ModeSymlink == os.ModeSymlink {
```

Comment:
This is also wrong (it has the same misunderstanding). This should be `st.Mode&unix.S_IFMT == unix.S_IFLNK`.